pages/learn/manage/billing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Billing
+---
+
+# Billing
+
+The billing page can be found in the dropdown at the top-right of the dashboard, directly beneath the account preferences. From this dashboard, you can update your information, change your payment plan, monitor your usage, and download invoices.
+
+## Account settings and usage
+
+At the top of the dashboard, you'll find a summary of your subscription with information on the total amount and due date for your next payment:
+
+<img src="/img/common/billing/sub_sum.png" width="100%">
+
+Beneath this you can see your current usage, including devices and collaborators:
+
+
+<img src="/img/common/billing/usage.png" width="80%">
+
+Beginning August 1st, 2018, your usage will be monitored to make sure you don't have more devices or collaborators than your plan allows. Any usage over your subscription limits will be [automatically added to your bill][autobill].
```

Comment:
let's remove this from here and address it all under the Automated Billing section.